master/reference/calicoctl/resources/policy.md

```diff
@@ -44,17 +44,17 @@ spec:
 | name | The name of the policy. |         | string |
 | annotations | Opaque key/value information to be used by clients. | | map |
 
-
 #### Spec
 
 | Field      | Description                                                                                                                                           | Accepted Values | Schema                | Default |
 |------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|-----------------------|---------|
-| order      | (Optional) Indicates priority of this policy, with lower order taking precedence.  No value indicates highest order (lowest precedence)               |                 | float                 |         |
+| order      | Indicates ordering of this policy: policies with lower order are applied earlier.  No value means that the policy is applied last.                    |                 | float                 |         |
 | selector   | Selects the endpoints to which this policy applies.                                                                                                   |                 | [selector](#selector) | all()   |
```

Comment:
I would recommend including backticks around all() to get it to look like code, e.g., `all()`